pre_commit_hooks/check_merge_conflict.py

```diff
@@ -7,6 +7,7 @@
 CONFLICT_PATTERNS = [
     b'<<<<<<< ',
     b'======= ',
+    b'||||||| ',
```

Comment:
would you like to add a failing test for this?  the testsuite passes because that particular string is part of a larger string which contains the other strings

Reply:
Sure! Think I can get to that later tonight

Reply:
@asottile I gave this a go, but am encountering some friction with running tests locally (test dependencies are missing from requirements-dev.test and I'm getting unexplicable tmpfile errors). The build is also telling me that there is something incorrect about either my understanding of the how the tests work. So not pursuing for now, but leaving this open for others!

Reply:
I think all you need to do is add to this list: https://github.com/pre-commit/pre-commit-hooks/pull/586/files#diff-6224b62678ea600729955336d2e3f48b0c90be48e8ea57dbefe54e7c00131532R106

are you using `tox` when testing locally?

Reply:
A bit confused there, that commit... is where I added to that list? Then CI fails on [this test](https://github.com/pre-commit/pre-commit-hooks/pull/586/commits/0c6a81a69f43717f79028f9938ca0d9d68ba8eeb#diff-6224b62678ea600729955336d2e3f48b0c90be48e8ea57dbefe54e7c00131532R136-R138) that I'd expect to succeed.

I've never used `tox` before but I figured it out (tho still have odd tmpfile-caused test failures). Some dev-instruction that you should be using it, and how to use, may help with future contributors.

Speaking of which, thanks for the help!

Reply:
yeah you shouldn't need to touch that test -- it's testing a different behaviour (when `--assume-in-merge` is passed it *should* check for those strings) -- I think you can revert your test changes and just add to the tests on line 106